### PR TITLE
[Docs] Japanese translation of docs/support.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add Japanese translation of docs/support.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/support.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  git diff 0.9.51 HEAD -- docs/support.md | cat
+-->
+
+QMK に関して助けを得るための多くのリソースがあります。
+
+コミュニティスペースに参加する前に[行動規範](https://qmk.fm/coc/)を読んでください。
+
+## リアルタイムチャット
+
+何かについて助けが必要な場合は、迅速なサポートを受けるための最良な場所は、[Discord Server](https://discord.gg/Uq7gcHh) です。通常は誰かがオンラインで、非常に助けになる多くの人がいます。
```
Comment (s-show) [inline docs/ja/support.md:14]: 「てにをは」レベルの細かい指摘で申し訳ありませんが、「最良な場所」より「最良の場所」の方が自然な日本語になる気がします。
```suggestion
何かについて助けが必要な場合は、迅速なサポートを受けるための最良の場所は、[Discord Server](https://discord.gg/Uq7gcHh) です。通常は誰かがオンラインで、非常に助けになる多くの人がいます。
```

Comment (umi-umi) [reply docs/ja/support.md]: ご指摘ありがとうございます。
見直しているうちにどんどん日本語が怪しくなっていって、深夜のテンションでコミットするという悪循環が。。
お手数をおかけしますが、どしどし指摘いただければ幸いですmm

**Review by shelaf — APPROVED** [review]

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
ありがとうございました！

**Pull request merged**